vic/vic_run/include/vic_def.h

```diff
@@ -33,6 +33,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <strings.h>
```

Comment:
In order to use the `-std=C99` (which guarantees a number of modules that we are depending on), I needed to add this. Anywhere we do a `strcasecmp` needs it? Do you think we should put it only in the source files that use it or is it okay to distribute globally like this?